button/popup.css

```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
 .footnote {
     width: 100%;
     height: 55px;
-    bottom: 0;
     color: #959595;
     padding-left: 15px;
     line-height: 57px;
     background: #fff;
+    float: left;
     cursor: default;
     border-top: 1px solid #e3e3e3;
 }
 
 .site { /* each note area */
     height: 50px; /* prev hieght 40px */
-    padding: 2px 0 2px 0;
+    margin: 2px 0 0px 0;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/1772214731/issues/source?bid=2884821&fileBranchId=3065795#l70)